src/systems/filecoin_mining/storage_mining/storage_miner_actor_code.go

```diff
@@ -598,12 +620,20 @@ func (a *StorageMinerActorCode_I) PreCommitSector(rt Runtime, info sector.Sector
 
 	st._assertSectorDidNotExist(rt, info.SectorNumber())
 
+	// @param dealIDs []deal.DealID info.DealIDs()
```

Comment:
ibid, see #677 for example of how to serialize args (in that case with randomness args)